k8s/k8s.go

```diff
@@ -2665,76 +2696,143 @@ func (k *k8sOps) ListEvents(namespace string, opts meta_v1.ListOptions) (*v1.Eve
 	return k.client.CoreV1().Events(namespace).List(opts)
 }
 
+func (k *k8sOps) CreateCRD(resources []CustomResource) error {
+	for _, resource := range resources {
+		err := k.createCRD(resource)
+		if err != nil {
+			return err
+		}
+	}
+
+	for _, resource := range resources {
+		if err := k.waitForCRDInit(resource); err != nil {
```

Comment:
We should have a separate method to validate/wait like we have for other objects instead of blocking the create call.

Reply:
done